benchmarks/matmul_benchmark/setup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash 
+
+currPath=$(dirname "$0")
+# python modules installation
+if ! pip install -r $currPath/requirements.txt
```

Comment:
I think the if is always true, in any case the pip install is always run

Reply:
That's right and correct: he is using `!` to detect any failures and to print "Required modules installed" if everything goes right. Rightfully the if condition is always ran.

`pip` should be always invoked and it should handle already available modules itself, therefore, even tho not the most elegant solution, the script logic is correct.